tests/update/updates_packagekit_gpk.pm

```diff
@@ -34,8 +34,8 @@ sub turn_off_screensaver {
 
 # Update with GNOME PackageKit Update Viewer
 sub run {
-    if (sle_version_at_least('15')) {
-        record_soft_failure 'bsc#1057223';    # updates_packagekit_gpk is disabled for SLE15 because of bsc#1061243
+    if (sle_version_at_least('15') && is_sle()) {
+        record_soft_failure 'bsc#1061243';    # updates_packagekit_gpk is disabled for SLE15 because of bsc#1061243
         return;
```

Comment:
That will fix it at least for me (good enough 😄) - but on SLE this will still fail, as an early return won't set $self->result values (assert_screen does it implicit)

add ```$self->result('ok');``` before returning

Reply:
thanks , for the tip :)